pkg/template/sample/report/BUILD

```diff
@@ -0,0 +1,51 @@
+package(default_visibility = ["//visibility:public"])
+
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
```

Comment:
we aren't using `go_test` here. nor are we using the `go_prefix` that follows, or `go_proto_library` in the rule after that.